spec/models/user_spec.rb

```diff
@@ -3,5 +3,69 @@
 require "rails_helper"
 
 RSpec.describe User, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  context "user attrs validation" do
+    let(:user_attributes) do
+      {
+        email: email,
+        password: password,
+        password_confirmation: password_confirmation
+      }
+    end
+    let(:email) { "valid_email@gmail.com" }
+    let(:password) { "123456" }
+    let(:password_confirmation) { password }
+    let(:user) { described_class.new(user_attributes) }
+    it "accepts valid params" do
+      expect(user.valid?).to be_truthy
+    end
+    context "role of the created user" do
+      it "accepts false" do
+        expect(user.admin?).to be_falsey
+      end
+    end
+    context "invalid params" do
+      context "invalid email" do
+        context "Domain missing" do
+          let(:email) { "test@example" }
+          it "accepts invalid params" do
+            expect(user.valid?).to be_falsey
+          end
+        end
+        context "Name missing" do
+          let(:email) { "@example.com" }
+          it "accepts invalid params" do
```

Comment:
same as above comment about 
`it "accepts invalid params"` but it doesn't
your test is about validation
and you are testing the outcome, not what it accepts